server/service/integration_core_test.go

```diff
@@ -6509,40 +6519,6 @@ func (s *integrationTestSuite) TestAppleMDMNotConfigured() {
 	s.Do("POST", "/api/latest/fleet/mdm/apple/dep/key_pair", nil, http.StatusOK)
 }
 
-func (s *integrationTestSuite) TestWindowsMDMNotConfigured() {
```

Comment:
@gillespi314 just to double-check this is correct - I had a merge conflict as in my PR I added my (windows MDM-related) endpoint to the existing list of MDM (Apple) endpoints and removed the Apple-specific naming as the common tests do not depend on the platform (they just check that the middleware correctly prevents access and returns the proper error).

I don't think we need distinct sets of endpoints by platform so I went ahead and removed the Windows-specific test and endpoints list that you added. I think ultimately the middleware for your (Windows+Apple) MDM endpoint would be a check that either Apple or Windows are enabled? And I believe that would be properly caught by the single list of mdm mw endpoints and tests.